README.md

```diff
@@ -276,3 +276,29 @@ class MyModule extends VuexModule {
   }
 }
 ```
+
+If you would like to preserve the state e.g when loading in the state from [vuex-persist](https://www.npmjs.com/package/vuex-persist)
+
+
+```diff
+...
+
+-- @Module({ dynamic: true, store: store, name: 'mm' })
+++ @Module({ dynamic: true, store: store, name: 'mm', preserveState: true })
```

Comment:
This looks like a diff of a diff... maybe a bad merge?

Reply:
Nah this was intentional, this is so that the user can see which line to change. Could be removed. I thought it made it more clear.
![Screenshot 2019-04-26 at 11 48 47](https://user-images.githubusercontent.com/1766076/56799454-49044b00-6819-11e9-98dd-70b268bb1658.png)